lib/profile/commands/configure.rb

```diff
@@ -53,7 +54,7 @@ def ask_questions
                 output = result.stdout.chomp
                 if !result.success?
                   smart_log.debug("Command '#{question.default_smart}' failed to run: #{result.stderr}")
-                elsif output.match(Regexp.new(question.validation.format))
+                elsif (!question.validation.has_key?(:format) || output.match(Regexp.new(question.validation.format)))
```

Comment:
This will pass the validation if the question has a format validator, _not_ if it passes said validator. We should be using `&&`, not `||`.

Reply:
If my understanding is correct, the first half is checking if it DOESN'T have a format validator (in which case it's safe to add it), while the second half is checking the validation (in the case that it does have one)

Reply:
Yup, that certainly is a `!` that I missed. Ignore me :sweat_smile: